File: 250720/정상적인 구구단/normal-multiple.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int intN = sc.nextInt();

        // 반복문
        for (int i = 1; i < intN + 1; i++) {
            for (int j = 1; j < intN + 1; j++) {
                if (j == intN) {
                    System.out.printf("%d * %d = %d", i, j, i*j);
                } else {
                    System.out.printf("%d * %d = %d, ", i, j, i*j);
                }
            }
            System.out.println("");
        }
    }
}

File: 250720/정상적인 구구단/normal-multiple.py
# 입력
intN = int(input())

# 반복문
for i in range(1, intN + 1, 1) :
    for j in range(1, intN + 1, 1) :
        if j == intN :
            print(f"{i} * {j} = {i*j}", end = " ")
        else :
            print(f"{i} * {j} = {i*j}", end = ", ")
    print("")